models/dish/dishModel.js

```diff
@@ -0,0 +1,77 @@
+import { Schema, model } from 'mongoose';
+
+const dishSchema = new Schema(
+  {
+    name: {
+      type: String,
+      required: [true, 'Name of the dish is required'],
+      unique: true,
+    },
+    owner: {
+      type: Schema.Types.ObjectId,
+      ref: 'chef',
+      required: [true, 'Owner (chef) ID is required'],
+    },
+    image: {
+      type: String,
+      required: [true, 'Image URL is required'],
+    },
+    description: {
+      type: String,
+      required: [true, 'Description of the dish is required'],
+    },
+    ingredients: {
+      type: Schema.Types.ObjectId,
+      ref: 'ingredient',
+      required: [true, 'Ingredient ID is required'],
+    },
+    price: {
+      type: Number,
+      required: [true, 'Price of the dish is required'],
+    },
+    isVegan: {
+      type: Boolean,
+      required: [
+        true,
+        'Information about the dish being vegan or not is required',
+      ],
+    },
+    cuisine: {
+      type: String,
```

Comment:
it's better to use enum here

Reply:
I opted for strings instead of enums in the schema to allow easy expansion of categories on the frontend without backend modifications. Ok, fixed.

Reply:
Maybe it's better to put it in a separate file? If we want to use it in many places?

Reply:
we can create an array (file with all such arrays) and make it accessible to front and back, then it will be the one source of true if we need to add something